extensions/notification/NotificationQueuePopulator.js

```diff
@@ -91,6 +91,18 @@ class NotificationQueuePopulator extends QueuePopulatorExtension {
         return false;
     }
 
+    /**
+     * Notification rules are normally verified when setting the notification
+     * configuration (even for wildcards), however we need an explicit check at
+     * this level to filter out non standard events that might be valid for one
+     * of the wildcard rules set.
+     * @param {String} eventType - notification event type
+     * @returns {boolean} - true if notification is supported
+     */
+    _isNotificationEventSupported(eventType) {
+        return supportedNotificationEvents.has(eventType);
+    }
```

Comment:
not really an issue, but we may (at some point, just maybe) want to allow explicitely registering for these non-standard notifications (even though we probably don't want those to be picked up automatically by wildcard configs indeed)
→ we should remember this, may trick us later... (but certainly fine for now)